packages/node_modules/@webex/plugin-memberships/src/memberships.js

```diff
@@ -415,7 +500,47 @@ const Memberships = SparkPlugin.extend({
       body: membership
     })
       .then((res) => res.body);
+  },
+
+  /**
+   * Updates the lastSeenId attribute of a membership
+   * @instance
+   * @memberof Mememberships
+   * @param {string} message
+   * @returns {Promise<MembershipObject>}
+   */
+  updateLastSeen(message) {
+    const activity = {
+      id: deconstructHydraId(message.id).id
+    };
+    const conversation = {
+      id: deconstructHydraId(message.roomId).id
+    };
+
+    return this.spark.internal.conversation.acknowledge(conversation, activity)
+      .then((activity) => {
+        console.log(activity);
```

Comment:
probably just for debugging

Reply:
Yes, thought I pulled those out before committing.   WIll update in next commit.

Reply:
Gone now.